README.md

```diff
@@ -1 +1,49 @@
-Microservice to handle exercises
+[![Continous Integration](https://github.com/OnlineTrainingPlatform/nodeTemplate/actions/workflows/continous_integration.yml/badge.svg)](https://github.com/OnlineTrainingPlatform/nodeTemplate/actions/workflows/continous_integration.yml)
```

Comment:
``OnlineTrainingPlatform/nodeTemplate`` should be ``OnlineTrainingPlatform/exercises``